app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -119,4 +119,23 @@ def remove_disks(disks, vm)
       disks.each { |disk_id| service.attachment_service(disk_id).remove }
     end
   end
+
+  def vm_migrate(vm, options = {})
```

Comment:
I was trying to make it similar to the vmware provider where it is also placed in the infra manager. Also, it needs to be called from the vm_migrate_workflow